packages/client/upgrade-guides/to-v0.12.0.md

```diff
@@ -0,0 +1,123 @@
+# Upgrading `@knocklabs/client` to v0.12.0
+
+This upgrade guide includes instructions for upgrading to the latest version of `@knocklabs/client` and the Knock SDK packages that depend on it.
+
+This update impacts the following packages:
+- Upgrading `@knocklabs/client` to v0.12.0
+- Upgrading `@knocklabs/react-core` to v0.5.0
+- Upgrading `@knocklabs/react` to v0.6.0
+- Upgrading `@knocklabs/react-native` to v0.6.0
+- Upgrading `@knocklabs/expo` to v0.3.0
+
+## Breaking changes
+
+As part of this update, we've upgraded our dependency on `zustand` to use the latest version. This means that we've had to make some breaking changes to the API around accessing the data store.
+
+### Changes to `feedClient.store`
+
+Consumers no longer need to use their own Zustand dependency to access the store. You can now access the store directly using the `store` property on the `feedClient` object.
```

Comment:
Nit: I think we should stay consistent using `you` when referring to the reader rather than referring to them as `Consumers` then swapping to directly speaking to them.